packages/ui/src/command.tsx

```diff
@@ -17,20 +17,48 @@ const Command = React.forwardRef<
 	<CommandPrimitive
 		ref={ref}
 		className={cn(
-			"flex h-full w-full flex-col overflow-hidden rounded-md bg-white text-gray-950",
+			"flex h-full w-full flex-col overflow-hidden bg-white text-gray-950",
```

Comment:
interesting: for some reason this "rounded" caused the whole command list to be blurry

https://github.com/hngngn/shadcn-solid/issues/100